source/user-manual/reference/ossec-conf/client-buffer.rst

```diff
@@ -54,10 +55,21 @@ events_per_second
 
 Specifies the number of events that can be sent to the manager per second.
 
++--------------------+----------------------------------------------------------------------+
+| **Default value**  | 500                                                                  |
++--------------------+----------------------------------------------------------------------+
+| **Allowed values** | Any number between 0 and 100000. Zero to disable the functionality.  |
```

Comment:
This can be confusing since users can understand that the buffer is disabled. I think it is better to include a note where explain that setting 0 EPS means that the buffer is still working but it will dispatch events as fast as possible.

Reply:
Done